sdk/inc/azure/core/az_version.h

```diff
@@ -17,19 +17,18 @@
 
 /// The version in string format used for telemetry following the `semver.org` standard
 /// (https://semver.org).
-#define AZ_SDK_VERSION_STRING "1.0.0"
+#define AZ_SDK_VERSION_STRING "1.1.0-beta.1"
```

Comment:
This change isn't necessary, it is covered by https://github.com/Azure/azure-sdk-for-c/pull/1341.